### DISCUSSION TITLE
HandyControls Globalization looks diffs from 3.0.0

### DISCUSSION THREAD

**Discussion by GF-Huang** [HandyOrg/HandyControl#699] [Q&A] [answered]
@ghost1372 In 3.0.0 use `LocalizationManager`, 3.1.1 use `ConfigHelper`, the [docs](https://hosseini.ninja/handycontrol/langs/) changes dramatically. Did I remember it wrong?

**Comment by ghost1372** [comment, marked as answer]
@GF-Huang 
I've changed how to change the language in version 3.1
In version 3.0, you had to use both LocalizationManager and ConfigHelper to change the language
But in version 3.1 only ConfigHelper is enough
https://hosseini.ninja/handycontrol/persianToolkit/dynamicLanguage/#How-to-Change-language-in-Runtime

**Reply by ghost1372** [reply]
Keep in mind that there are different methods for multilingualism
This link is for the original version https://hosseini.ninja/handycontrol/langs/
and this link is for handycontrols
https://hosseini.ninja/handycontrol/persianToolkit/dynamicLanguage/

**Reply by GF-Huang** [reply]
Oh, the original version https://hosseini.ninja/handycontrol/langs/ fakes me.
